README.md

```diff
@@ -156,7 +156,7 @@ Props for the spinner button
 | Props                 | Default |          Type           | Description                                                                                          |
 | :-------------------- | :-----: | :---------------------: | :--------------------------------------------------------------------------------------------------- |
 | **onPress** |    -    |   function  | The function to execute upon tapping the button |
-| animationType |    null or undefined    |         string          | Type of animation for the button and spinner, For more details about properties, refer [react-native-animatable](https://www.npmjs.com/package/react-native-animatable)   |
+| animationType |    null or undefined    |         string          | Type of animation for the button and spinner, For more details about properties, refer [react-native-animatable](https://www.npmjs.com/package/react-native-animatable) and also support "ripple-effect"  |
```

Comment:
@rajanpayal-simform we should revise this description for animationType
e.g Animation type for SpinnerButton, Currently supports ripple effect only